### DISCUSSION TITLE
Wlan Connection Dialog if Pi dont find an Connection

### DISCUSSION THREAD

**Discussion by jeffreyx185** [guysoft/FullPageOS#536] [Ideas]
Hi im Using FullpageOs as a Kiosk in different Locations. 
So The Wifi SSID and PW is changing a lot.

Is it possible to open a Wifi Dialog if the Pi cannot find a active connection?

Maybe one of you have a tip how it can be programmed?

Greetings

**Comment by guysoft** [comment]
You should bw able to set multiple ssids in the wpa-supplicant file and it should connect to the first one.

The behavior might have changed in book room if anyone wants to test
